.github/workflows/cli-features.yml

```diff
@@ -0,0 +1,26 @@
+# Tests extra CLI features as and their dependency with core features.
```

Comment:
I think there's a typo in this comment around _features as and their dependency_